.buildkite/pipeline.yml

```diff
@@ -163,25 +111,24 @@ steps:
       - rm -rf "/Users/administrator/Library/Logs/Unreal Engine/LocalBuildLogs/*"
       - features/scripts/build-fixture.sh Mac
     timeout_in_minutes: 90
-    key: mac_fixture
+    key: mac_fixture_4_27
 
-  ##############################################################################
   #
-  # Run E2E tests
+  # E2E Tests
   #
-  ##############################################################################
 
-  - label: 'E2E Tests - Android 11'
+  # UE 4.27
+  - label: 'E2E Tests - Android 11 - UE 4.27'
     depends_on:
-      - android_fixture
+      - android_fixture_4_27
```

Comment:
The inconsistency with other steps makes me a little twitchy.  I'd make all of them look like:
```
  depends_on: some_step
```
(and without the `[]`s on other steps).